src/prospectScraper/Program.cs

```diff
@@ -39,720 +33,27 @@ static void Main(string[] args)
-                    mdpList.Add(mockDraftPick);
-                }
-            }
-            return mdpList;
-        }
-
-        public static MockDraftPick CreateMockDraftEntry(HtmlNode tableRow, string pickDate)
-        {
-            var childNodes = tableRow.ChildNodes;
-            var node1 = childNodes[1].InnerText; //pick number?
-            string pickNumber = node1.Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .Replace(" ", "");
-            var node3 = childNodes[3]; //team (and team image)?
-            var teamCity = node3.ChildNodes[0].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .TrimEnd();
-            var node5 = childNodes[5]; //Has Child Nodes - Player, School, Position, Reach/Value
-            string playerName = node5.ChildNodes[1].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .TrimEnd();
-            string playerSchoolBeforeChecking = node5.ChildNodes[3].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .TrimEnd(); // this may have a space afterwards.
-            string playerSchool = CheckSchool(playerSchoolBeforeChecking);
-            string playerPosition = node5.ChildNodes[5].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .Replace(" ", "");
-            string reachValue = node5.ChildNodes[9].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .Replace(" ", "");
-
-
-            MockDraftPick mdp = new MockDraftPick(pickNumber, teamCity, playerName, playerSchool, playerPosition, reachValue, pickDate);
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Mock Draft Round: " + mdp.round + Environment.NewLine);
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Pick Number: " + mdp.pickNumber + Environment.NewLine);
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Player: " + mdp.playerName + Environment.NewLine);
-            return mdp;
-        }
-
-        public static DateTime ChangeDateStringToDateTime(string scrapedDate, bool parseDate)
```

Comment:
This method is new. It needs to be in ProspectScraper.cs. 

It replaces the functionality of GetDraftDate() and FormatDraftDate()

Reply:
Yeah, this got hosed in the merge conflict resolution. I'm adding it back.